vulture.py

```diff
@@ -165,8 +167,14 @@ def _get_modules(self, paths, toplevel=True):
                 sys.exit('Error: %s could not be found.' % path)
         return modules
 
+    def _get_stdlib_whitelist_file(self):
+        script = os.path.abspath(stdlib.__file__)
```

Comment:
I think it's best to turn vulture into a package first, before we include the default whitelist. Can you open an issue for this? If you want to start a pull request, we need the following layout:

```
vulture/
    __init__.py (contains __version__)
    core.py (with the contents from old vulture.py)
```